src/kademlia/dht_tracker.cpp

```diff
@@ -291,6 +291,8 @@ namespace libtorrent { namespace dht
 		}
 
 		if (size <= 20 || *buf != 'd' || buf[size-1] != 'e') return false;
+		// remove this line/check once the DHT supports IPv6
+		if (!ep.address().is_v4()) return false;
```

Comment:
@arvidn Not sure if return true of false here, if it's better to return true, then maybe this line should be below the following `inc_stats_counter`s